modules/knative-serving-CR-system-deployments.adoc

```diff
@@ -6,13 +6,14 @@
 [id="knative-serving-CR-system-deployments_{context}"]
 = Overriding system deployment configurations
 
-You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR). Currently, overriding default configuration settings is supported for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields.
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR). Currently, overriding default configuration settings is supported for the `resources`, `replicas`, `labels`, `annotations`, and `nodeSelector` fields.
```

Comment:
The full list of overrides is [here](https://github.com/openshift-knative/serverless-operator/blob/a1ba71728d05ed7b307a2612cd10ac3ede3dedb6/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/common.go#L220-L251).
Includes also tolerations and affinity, do we reference those somewhere else?

Reply:
I think that's out of the scope of this PR tbh, the intention was just to add resources.
Feel free to open an additional Jira if there are others missing (please provide examples for each).